internal/cli/runner_agent.go

```diff
@@ -129,7 +135,14 @@ func (c *RunnerAgentCommand) Run(args []string) int {
 }
 
 func (c *RunnerAgentCommand) Flags() *flag.Sets {
-	return c.flagSet(0, nil)
+	return c.flagSet(flagSetOperation, func(set *flag.Sets) {
+		f := set.NewSet("Command Options")
+		f.BoolVar(&flag.BoolVar{
+			Name:   "enable-dynamic-config",
+			Target: &c.flagDynConfig,
```

Comment:
Oops, didn't hit save on this comment when I hit submit on my review....

Should we be explicit and default to false here, or is unset and nil essentially false?

Reply:
Unset is false by default here, yeah.